install-scripts/install_ipfix-rita.sh

```diff
@@ -0,0 +1,541 @@
+#	fi
+
+#	return 0
+# } #End of do_system_tests
+
+usage_text () {
+	cat >&2 <<EOHELP
+This script will install ipfix-rita.
+On the command line, enter one of the following:
+$0
+$0 rita ip.address.for.rita
+The IP address can be 127.0.0.1 to indicate a local RITA installation.
+EOHELP
+	exit 1
+}
+
+validated_ssh_target () {
+	#Return an ssh target that we've confirmed we can reach.  "$2" is the initial target to try, and may be replaced.
+	#The potential target - both as supplied as parameter 2 and as returned at the end - may be blank, indicating "do not install this"
+	target_type="$1"
+	potential_target="$2"
+
+	echo 'About to execute commands on '"$potential_target"' .  You may be prompted one or more times for the password for this system.' >&2
+	echo 'Note: if you are using an ssh client other than openssh and have not set up ssh key access to this system, you may be prompted for your password multiple times.' >&2
+	while [ -n "$potential_target" ] && ! can_ssh "$potential_target" ; do
+		echo "What is the hostname or IP address where you would like to send IPFIX logs to?  Enter 127.0.0.1 if you want to run on this system.  Press Ctrl-C to exit." >&2
+		read potential_target <&2
+		if [ -n "$potential_target" -a "$potential_target" != '127.0.0.1' ]; then
+			echo "Do you access that system with a username different than ${USER}?  If so, enter that username now, otherwise just press enter if your account is ${USER} on the remote system too." >&2
+			read potential_user <&2
+			if [ -n "$potential_user" ]; then
+				potential_target="${potential_user}@${potential_target}"
+			fi
+		fi
+	done
+
+	echo "$potential_target"
+} #End of validated_ssh_tartget
+
+get_line() {
+	rita_file=$1
+	search_key=$2
+
+	#get the line that matches the search key that isn't commented out
+	line=`cat $rita_file | grep $search_key`
+	line=`echo $line | cut -d ' ' -f 2`
+
+	echo $line
+} #End of get line
+
+get_rita_data_interactive() {
+	IPFIX_RITA_NETWORK_GATEWAY=$(docker inspect ipfix_rita_default --format "{{with (index .IPAM.Config 0)}}{{.Gateway}}{{end}}")
+	RITA_MONGO_URI="mongodb://$IPFIX_RITA_NETWORK_GATEWAY:27017"
+
+	echo "" >&2
+	echo "IPFIX-RITA needs to write to a MongoDB database controlled by RITA." >&2
+	echo "By default, this installer assumes RITA and MongoDB are installed on the Docker host." >&2
+	echo "In order to support this type of installation, you will need to" >&2
+	echo "add the suggested Docker interface below to the list of bindIP's in /etc/mongod.conf." >&2
+	echo "If needed, please do so, and restart MongoDB before continuing." >&2
+	echo "Restart MongoDB with \$ sudo systemctl restart mongod.service" >&2
+	echo "Note: the default configuration is not recommended. IPFIX-RITA will likely perform" >&2
+	echo "better if it is installed on a machine separate from RITA/ MongoDB." >&2
+	echo "" >&2
+
+	echo "What MongoDB URI should IPFIX-RITA use to contact the RITA database [$RITA_MONGO_URI]: " >&2
+	read RITA_MONGO_URI <&2
```

Comment:
This deletes the default RITA_MONGO_URI for the docker host.

The original code reads the response into the default REPLY variable to avoid this.

```
  read -p "What MongoDB URI should IPFIX-RITA use to contact the RITA database [\$RITA_MONGO_URI]: " -r
  if [ -n "\$REPLY" ]; then
    RITA_MONGO_URI="\$REPLY"
  fi
```